src/o2.cpp

```diff
@@ -596,7 +596,8 @@ void O2::onRefreshFinished() {
         else
         {
           setToken(tokens.value(O2_OAUTH2_ACCESS_TOKEN).toString());
-          setExpires(QDateTime::currentMSecsSinceEpoch() / 1000 + static_cast<qint64>(tokens.value(O2_OAUTH2_EXPIRES_IN).toInt()));
+          const int expiresIn = tokens.value(O2_OAUTH2_EXPIRES_IN).toInt();
+          setExpires(QDateTime::currentMSecsSinceEpoch() / 1000 + expiresIn);
```

Comment:
I know it's not strictly required, but let's leave this here to make things nice and explicit:

```suggestion
          setExpires(QDateTime::currentMSecsSinceEpoch() / 1000 + static_cast<qint64>(expiresIn));
```

Reply:
Then again, `1000` is also promoted implicitly...